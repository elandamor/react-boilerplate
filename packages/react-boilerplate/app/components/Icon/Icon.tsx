import React, { SFC } from 'react';
import classNames from 'classnames';
// Dependencies
import ICONS from './constants';
// Styles
import Wrapper from './styles';

/**
 * @render react
 * @name Icon component
 * @description Icon component.
 * @example
 * <Icon
 *  icon="star"
 * />
 */

interface IProps {
  className?: string;
  icon: string;
  size?: number;
  viewBox?: string;
}

const Icon: SFC<IProps> = ({ className, icon, size, viewBox }) => (
  <Wrapper className={classNames('c-icon', className)} aria-hidden="true">
    <svg height={`${size}`} width={`${size}`} viewBox={`${viewBox}`}>
      {ICONS[icon] &&
        ICONS[icon]
          .split(',')
          .map((p: string, idx: number) => <path key={idx} d={p} />)}
    </svg>
  </Wrapper>
);

Icon.defaultProps = {
  size: 24,
  viewBox: '0 0 24 24',
};

export default Icon;
