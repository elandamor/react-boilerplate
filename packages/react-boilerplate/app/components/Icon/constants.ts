export default {
  add: 'M11 5v6H5v2h6v6h2v-6h6v-2h-6V5h-2z',
  add_user:
    'M14 0c-2.21 0-4 1.79-4 4s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zM3 4v1c0 .552-.448 1-1 1H1c-.552 0-1 .448-1 1s.448 1 1 1h1c.552 0 1 .448 1 1v1c0 .552.448 1 1 1s1-.448 1-1V9c0-.552.448-1 1-1h1c.552 0 1-.448 1-1s-.448-1-1-1H6c-.552 0-1-.448-1-1V4c0-.552-.448-1-1-1s-1 .448-1 1zm11 6c-6.102 0-8 4-8 4 0 1.105.895 2 2 2h12c1.105 0 2-.895 2-2 0 0-1.898-4-8-4z',
  advance:
    'M 17.21875 5.28125 L 15.78125 6.71875 L 20.0625 11 L 12 11 L 12 13 L 20.0625 13 L 15.78125 17.28125 L 17.21875 18.71875 L 23.21875 12.71875 L 23.90625 12 L 23.21875 11.28125 L 17.21875 5.28125 z M 0 11 L 0 13 L 2 13 L 2 11 L 0 11 z M 4 11 L 4 13 L 6 13 L 6 11 L 4 11 z M 8 11 L 8 13 L 10 13 L 10 11 L 8 11 z',
  back:
    'M14.781 2.281l-9 9-.687.719.687.719 9 9 1.438-1.438L7.937 12l8.282-8.281z',
  bookmark:
    'M.5 17.242l5.515-2.364c.63-.27 1.34-.27 1.97 0l5.515 2.364V2c0-.824-.676-1.5-1.5-1.5H2C1.18.5.51 1.173.51 2L.5 17.242z',
  breadcrumb:
    'M1.575.7l5.797 5.797c.27.28.27.725 0 1.006L1.575 13.3c-.278.278-.728.278-1.006 0-.28-.278-.28-.728 0-1.006l4.94-4.94c.194-.196.194-.512 0-.708L.57 1.706C.29 1.428.29.978.57.7c.277-.278.727-.278 1.005 0z',
  camera:
    'M6 1c-.64.64-1.51 1-2.414 1H2C.898 2 0 2.898 0 4v12c0 1.102.898 2 2 2h16c1.102 0 2-.898 2-2V4c0-1.102-.898-2-2-2h-1.586C15.51 2 14.64 1.64 14 1c-.64-.64-1.51-1-2.414-1H8.414C7.51 0 6.64.36 6 1zm4 4c2.8 0 5 2.2 5 5s-2.2 5-5 5-5-2.2-5-5 2.2-5 5-5zm0 1.5c-1.934 0-3.5 1.566-3.5 3.5s1.566 3.5 3.5 3.5 3.5-1.566 3.5-3.5-1.566-3.5-3.5-3.5z',
  cancel:
    'M10 0C4.5 0 0 4.5 0 10s4.5 10 10 10 10-4.5 10-10S15.5 0 10 0zm4.2 14.2c-.387.385-1.013.385-1.4 0l-2.093-2.094c-.39-.39-1.024-.39-1.414 0L7.2 14.2c-.387.385-1.013.385-1.4 0-.385-.387-.385-1.013 0-1.4l2.094-2.093c.39-.39.39-1.024 0-1.414L5.8 7.2c-.385-.387-.385-1.013 0-1.4.387-.385 1.013-.385 1.4 0l2.093 2.094c.39.39 1.024.39 1.414 0L12.8 5.8c.387-.385 1.013-.385 1.4 0 .385.387.385 1.013 0 1.4l-2.094 2.093c-.39.39-.39 1.024 0 1.414L14.2 12.8c.385.387.385 1.013 0 1.4z',
  close:
    'M5.625 4.219L4.219 5.625 10.594 12l-6.375 6.375 1.406 1.406L12 13.406l6.375 6.375 1.406-1.406L13.406 12l6.375-6.375-1.406-1.406L12 10.594 5.625 4.219z',
  country:
    'M17 0C7.626 0 0 7.626 0 17c0 9.374 7.626 17 17 17 9.374 0 17-7.626 17-17 0-9.374-7.626-17-17-17zm14.094 15.583h-4.217C26.674 11.22 25.552 7.38 23.833 4.6c3.976 2.198 6.79 6.25 7.261 10.984zm-15.51-12.45v12.45h-5.63c.32-6.097 2.669-11.191 5.63-12.45zm0 15.284v12.45c-2.961-1.259-5.31-6.353-5.63-12.45h5.63zm2.833 12.45v-12.45h5.666c-.32 6.097-2.706 11.191-5.666 12.45zm0-15.284V3.133c2.96 1.259 5.31 6.353 5.63 12.45h-5.63zM10.167 4.6c-1.72 2.78-2.841 6.621-3.044 10.984H2.906C3.377 10.85 6.19 6.797 10.167 4.6zM2.906 18.416h4.217c.203 4.363 1.325 8.204 3.044 10.984-3.976-2.198-6.79-6.25-7.261-10.984zM23.833 29.4c1.72-2.78 2.841-6.621 3.044-10.984h4.217c-.471 4.734-3.285 8.786-7.261 10.984z',
  delete:
    'M4.5.5c-.32.32-.754.5-1.207.5H1c-.552 0-1 .448-1 1s.448 1 1 1h12c.552 0 1-.448 1-1s-.448-1-1-1h-2.293C10.254 1 9.82.82 9.5.5 9.18.18 8.746 0 8.293 0H5.707C5.254 0 4.82.18 4.5.5zM1 5v12c0 1.102.898 2 2 2h8c1.102 0 2-.898 2-2V5c0-.552-.448-1-1-1H2c-.552 0-1 .448-1 1z',
  expand_arrow:
    'M1.02.687c-.413.434-.413 1.116 0 1.55l8.522 8.933c.407.408 1.068.41 1.477.002v-.002l8.52-8.933c.414-.434.414-1.116 0-1.55-.39-.41-1.037-.424-1.445-.034-.012.01-.023.022-.034.034l-6.334 6.64c-.762.8-2.028.83-2.827.067l-.067-.067L2.5.687C2.11.277 1.463.263 1.055.653c-.012.01-.023.022-.034.034z',
  explore:
    'M12 0C5.398 0 0 5.398 0 12s5.398 12 12 12 12-5.398 12-12S18.602 0 12 0zm0 1c6.05 0 11 4.95 11 11s-4.95 11-11 11S1 18.05 1 12 5.95 1 12 1zm4.602 5.724l-6.226 3.224c-.184.095-.333.244-.428.428l-3.224 6.226c-.127.246-.03.547.214.674.144.075.315.075.46 0l6.226-3.224c.184-.095.333-.244.428-.428l3.224-6.226c.127-.246.03-.547-.214-.674-.144-.075-.315-.075-.46 0zM12 10.5c.825 0 1.5.675 1.5 1.5s-.675 1.5-1.5 1.5-1.5-.675-1.5-1.5.675-1.5 1.5-1.5z',
  filter:
    'M11 2c-1.293 0-2.395.844-2.813 2H2v2h6.188C8.604 7.156 9.707 8 11 8s2.395-.844 2.813-2H22V4h-8.188C13.396 2.844 12.293 2 11 2zm0 2c.563 0 1 .438 1 1 0 .563-.438 1-1 1-.563 0-1-.438-1-1 0-.563.438-1 1-1zm6 5c-1.293 0-2.395.844-2.813 2H2v2h12.188c.417 1.156 1.519 2 2.812 2s2.395-.844 2.813-2H22v-2h-2.188C19.395 9.844 18.294 9 17 9zm0 2c.563 0 1 .438 1 1 0 .563-.438 1-1 1-.563 0-1-.438-1-1 0-.563.438-1 1-1zM7 16c-1.293 0-2.395.844-2.813 2H2v2h2.188c.417 1.156 1.519 2 2.812 2s2.395-.844 2.813-2H22v-2H9.812C9.396 16.844 8.293 16 7 16zm0 2c.563 0 1 .438 1 1 0 .563-.438 1-1 1-.563 0-1-.438-1-1 0-.563.438-1 1-1z',
  heart:
    'M14.398 0c-.916 0-1.99.45-3.22 1.346-.702.513-1.654.513-2.357 0C7.59.45 6.52 0 5.605 0 2.5 0 0 2.5 0 5.602c0 2.838 3.123 6.8 9.368 11.888.368.3.896.3 1.264 0C16.877 12.403 20 8.44 20 5.602 20 2.5 17.5 0 14.398 0z',
  home:
    'M18 17v-7c0-.55228.44772-1 1-1 .21193 0 .38374-.1718.38374-.38374 0-.10878-.04618-.21245-.12703-.28522L10.66898.60207c-.3803-.34228-.95762-.34228-1.33792 0L.7433 8.33104c-.15753.14177-.1703.3844-.02853.54193C.78754 8.95383.89122 9 1 9c.55228 0 1 .44772 1 1v7c0 .6.4 1 1 1h14c.6 0 1-.4 1-1z',
  globe:
    'M10 0C4.5 0 0 4.5 0 10s4.5 10 10 10 10-4.5 10-10S15.5 0 10 0zm0 1.594c1.102 0 2.156.2 3.125.593-.023.266-.23.555-.313.72-.1.3-.324.706-.624.905-.2.2-.48.49-.782.688-.3.2-.605.488-.906.688-.2.1-.305.218-.406.218H10c-.2-.2-.188-.7-.188-1 0-.5.29-.812.188-1.312 0-.102-.094-.906-.094-.906-.398-.102-.82 0-.72-.5.005-.016.025-.02.033-.032.26-.023.51-.062.78-.062zm-3.844 1.5c.305-.008.594.02.844.094.3.1.613.3.813.5.2.2.5.51.5.812 0 .102-.024.086-.126.188-.1.1-.18.125-.28.125-.302 0-.614-.024-.813-.125l-.407-.375c-.6-.2-1.492 1.1-1.593 1.5-.102.3-.094.992.406 1.093.3 0 .887-.613 1.188-.812.2-.102.3-.18.5-.282.8-.1 1.425.583 1.625 1.282.3.8-1.21 1.394-1.813 1.593-.2.102-.3-.1-.5 0-.5.2-1.094.805-1.094 1.407-.1.5-.117 1-.218 1.5-.102 0-.188.008-.188-.094v-.188c0-.3-.105-.71-.406-.812-.102-.102-.086-.094-.188-.094-.3 0-.605-.293-.906-.094-.2.2-.406.48-.406.782 0 .1-.008.21.094.312.2.102.425-.1.624 0 .2 0 .188.18.188.28.2.302.387.712.688.814h2c.3.102.8.207 1 .406.1.2.117.395.218.594.5.5 1.086.488 1.688.688.2.1.312.207.312.406.102.3.008.707-.094.906-.1.3-.207.707-.406.906-.2.2-.394.305-.594.407-.3.2-.51.574-.812.875 0 .05-.105.242-.22.406-.76-.21-1.495-.532-2.155-.938-.023-.152-.03-.3-.03-.343-.103-.3-.18-.7-.282-1-.2-.5-.625-1.024-.625-1.626 0-.5.113-.976-.188-1.374-.398-.5-1.195-.512-1.594-.813-.47-.313-.926-.758-1.25-1.22-.023-.26-.062-.514-.062-.78 0-2.715 1.343-5.148 3.375-6.688.39-.117.8-.21 1.182-.218zm9.125.406c1.896 1.547 3.13 3.898 3.13 6.5 0 2.777-1.386 5.277-3.5 6.813-.1-.32-.242-.64-.312-1-.2-.4-.305-.82-.406-1.22-.102-.3-.075-.792-.376-1.093-.1-.2-.3-.21-.5-.313-.7-.3-1.624.106-2.124-.593-.2-.3-.096-.602-.096-1 .2-.3.3-.606.5-.906.2-.5.492-.876.594-1.376.1-.2.022-.52.123-.718.104-.102.3-.18.5-.28.104-.103.19-.12.19-.22 0-.102-.093-.086-.093-.188 0-.1.086-.304.188-.406.5-.5 1-.906 1.5-1.406.1 0 .094-.094.094-.094.1-.102.005-.188-.096-.188-.2-.1-.3-.117-.5-.218-.102 0-.188.008-.188-.094-.1-.102-.008-.305.094-.406l.594-.907c.195-.195.402-.574.687-.687z',
  message:
    'M12.5,2C7.806,2,4,5.806,4,10.5c0,4.694,3.806,8.5,8.5,8.5c0.167,0,0.5,0,0.5,0v3c0,0,8-4.042,8-11.5 C21,5.806,17.194,2,12.5,2z',
  messages:
    'M11.207 1.235C10.352.917 9.44.75 8.5.75 4.22.75.75 4.22.75 8.5c0 4.28 3.47 7.75 7.75 7.75.69 0 1.25.56 1.25 1.25v1.183l.088-.057c.9-.595 1.802-1.302 2.64-2.117.502-.49.965-1 1.383-1.536-2.87-2.12-4.85-4.947-4.85-8.487 0-2.013.815-3.89 2.2-5.256zm7.38 12.438l-.11-.414c-.16-.59.19-1.198.78-1.356 2.814-.754 4.485-3.648 3.73-6.462-.753-2.814-3.647-4.484-6.46-3.73-2.816.754-4.486 3.647-3.732 6.46.48 1.795 1.66 3.21 3.344 4.296.772.5 1.61.9 2.443 1.202z',
  more_vert:
    'M12 3a1.999 1.999 0 1 0 0 4 1.999 1.999 0 1 0 0-4zm0 7a1.999 1.999 0 1 0 0 4 1.999 1.999 0 1 0 0-4zm0 7a1.999 1.999 0 1 0 0 4 1.999 1.999 0 1 0 0-4z',
  new_message:
    'M11.207 1.235C10.352.917 9.44.75 8.5.75 4.22.75.75 4.22.75 8.5c0 4.28 3.47 7.75 7.75 7.75.69 0 1.25.56 1.25 1.25v1.183l.088-.057c.9-.595 1.802-1.302 2.64-2.117.502-.49.965-1 1.383-1.54-2.87-2.12-4.85-4.948-4.85-8.488 0-2.013.82-3.89 2.2-5.256v.004zm6.08 1.48v2.57c0 .553-.45 1-1 1h-2.573c-.394 0-.714.32-.714.715 0 .394.32.714.714.714h2.572c.552 0 1 .448 1 1v2.572c0 .394.32.714.714.714.394 0 .714-.32.714-.714V8.714c0-.552.448-1 1-1h2.572c.394 0 .714-.32.714-.714 0-.394-.32-.714-.714-.714h-2.572c-.552 0-1-.448-1-1V2.714c0-.394-.32-.714-.714-.714-.394 0-.714.32-.714.714z',
  new_message_v2:
    'M10.34.487C9.494.167 8.592 0 7.663 0 3.43 0 0 3.483 0 7.78c0 4.295 3.43 7.778 7.663 7.778.682 0 1.236.562 1.236 1.255V18l.084-.057c.89-.597 1.782-1.307 2.61-2.125.497-.492.954-1.004 1.368-1.542-2.838-2.128-4.796-4.965-4.796-8.52 0-2.02.806-3.903 2.175-5.274l-.002.005zm5.48 1.77v1.765c0 .553-.447 1-1 1h-1.708c-.552 0-1 .448-1 1v.51c0 .552.448 1 1 1h1.708c.553 0 1 .447 1 1v1.764c0 .552.448 1 1 1h.472c.552 0 1-.448 1-1V8.532c0-.553.448-1 1-1H21c.552 0 1-.448 1-1v-.51c0-.552-.448-1-1-1h-1.708c-.552 0-1-.447-1-1V2.258c0-.552-.448-1-1-1h-.472c-.552 0-1 .448-1 1z',
  notification:
    'M 12 2 C 11.2 2 10.5 2.7 10.5 3.5 L 10.5 4.1875 C 7.3424666 4.8700668 5 7.6215005 5 11 L 5 15.5 L 3 17.8125 L 3 19 L 21 19 L 21 17.8125 L 19 15.5 L 19 11 C 19 7.6215005 16.657533 4.8700668 13.5 4.1875 L 13.5 3.5 C 13.5 2.7 12.8 2 12 2 z M 10 20 C 10 21.1 10.9 22 12 22 C 13.1 22 14 21.1 14 20 L 10 20 z',
  overview_pages:
    'M 2 2 L 2 4 L 22 4 L 22 2 L 2 2 z M 4 6 C 2.9 6 2 6.9 2 8 L 2 10 C 2 11.1 2.9 12 4 12 L 6 12 C 7.1 12 8 11.1 8 10 L 8 8 C 8 6.9 7.1 6 6 6 L 4 6 z M 10 6 L 10 8 L 21 8 L 21 6 L 10 6 z M 10 10 L 10 12 L 19 12 L 19 10 L 10 10 z M 4 15 C 2.9 15 2 15.9 2 17 L 2 19 C 2 20.1 2.9 21 4 21 L 6 21 C 7.1 21 8 20.1 8 19 L 8 17 C 8 15.9 7.1 15 6 15 L 4 15 z M 10 15 L 10 17 L 21 17 L 21 15 L 10 15 z M 10 19 L 10 21 L 19 21 L 19 19 L 10 19 z',
  private:
    'M8 0C4.676 0 2 2.676 2 6v1C.898 7 0 7.898 0 9v10c0 1.102.898 2 2 2h12c1.102 0 2-.898 2-2V9c0-1.102-.898-2-2-2V6c0-3.324-2.676-6-6-6zm0 2c2.277 0 4 1.723 4 4v1H4V6c0-2.277 1.723-4 4-4zm0 10c1.102 0 2 .898 2 2 0 1.102-.898 2-2 2-1.102 0-2-.898-2-2 0-1.102.898-2 2-2z',
  reset:
    'M18.146.76L16.47 2.437C14.706.942 12.472 0 10 0 4.465 0 0 4.465 0 10s4.465 10 10 10 10-4.465 10-10h-2c0 4.465-3.535 8-8 8s-8-3.535-8-8 3.535-8 8-8c1.922 0 3.66.695 5.063 1.844L13.76 5.146c-.195.196-.195.512 0 .708.094.093.22.146.353.146H18.5c.276 0 .5-.224.5-.5V1.113c0-.276-.224-.5-.5-.5-.133 0-.26.053-.354.147z',
  sidebar_toggle:
    'M19.36 12l-9.18 9.18L13 24l12-12L13 0l-2.82 2.82L19.36 12zM.18 0v24h4V0h-4z',
  scope:
    'M2.833 28.333H25.5V8.5H2.833v19.833zM0 31.167h28.333v-25.5H0v25.5z,M34 26.917h-2.833V2.833H4.25V0H34v26.917,M7.083 17H21.25v-2.833H7.083zM5.667 22.667h17v-2.834h-17z',
  sector:
    'M15.776 16.278h2.72V6.805h-2.72z,M28.016 19.04H6.256V5.44h-2.72v16.276h27.2V5.44h-2.72v13.6M32.096 0H2.176a1.36 1.36 0 1 0 0 2.72h29.92a1.36 1.36 0 1 0 0-2.72M18.496 25.16v-2.04h-2.72v2.04l-4.896 3.672a1.359 1.359 0 1 0 1.632 2.176l4.624-3.468 4.624 3.468a1.356 1.356 0 0 0 1.904-.272 1.359 1.359 0 0 0-.272-1.904l-4.896-3.672,M10.336 16.278h2.72V12.24h-2.72zM21.216 16.278h2.72V9.52h-2.72z',
  send: 'M 2 3 L 2 10.8125 L 18 12 L 2 13.1875 L 2 21 L 22 12 L 2 3 z',
  services:
    'M9.917 28.333h14.166V25.5H9.917v2.833zm-2.834 2.834h19.834v-8.5H7.083v8.5zM9.917 17h14.166v-2.833H9.917V17zm19.827-2.833h-2.827v-2.834H7.083v8.5h19.834V17h2.827c.758 0 1.423.704 1.423 1.506v5.577c0 .78-.635 1.417-1.417 1.417v2.833a4.255 4.255 0 0 0 4.25-4.25v-5.577c0-2.353-1.95-4.34-4.256-4.34zm-19.827-8.5h14.166V2.833H9.917v2.834zm-5.667 0h2.833V8.5h19.834V0H7.083v2.833H4.25A4.254 4.254 0 0 0 0 7.083v5.667A4.255 4.255 0 0 0 4.25 17v-2.833a1.418 1.418 0 0 1-1.417-1.417V7.083c0-.782.635-1.416 1.417-1.416z',
  star:
    'M18.2,21L12,17.3L5.8,21l1.6-7L2,9.3l7.2-0.6L12,2l2.8,6.6L22,9.3L16.6,14L18.2,21z',
  tick:
    'M19.281 5.281L9 15.563 4.719 11.28 3.28 12.72l5 5 .719.687.719-.687 11-11z',
  tune:
    'M5 3c-.55 0-1 .45-1 1v1H2v2h2v1c0 .55.45 1 1 1h1c.55 0 1-.45 1-1V4c0-.55-.45-1-1-1zm3 2v2h14V5zm9 4c-.55 0-1 .45-1 1v1H2v2h14v1c0 .55.45 1 1 1h1c.55 0 1-.45 1-1v-4c0-.55-.45-1-1-1zm3 2v2h2v-2zm-10 4c-.55 0-1 .45-1 1v1H2v2h7v1c0 .55.45 1 1 1h1c.55 0 1-.45 1-1v-4c0-.55-.45-1-1-1zm3 2v2h9v-2z',
};
