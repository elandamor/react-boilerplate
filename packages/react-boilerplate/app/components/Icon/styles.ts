import styled from 'styled-components';

const Wrapper = styled.i`
  margin: 0 auto;
  position: relative;

  svg {
    vertical-align: inherit;
  }
`;

export default Wrapper;
