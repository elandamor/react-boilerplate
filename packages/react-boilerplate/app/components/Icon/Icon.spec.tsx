// Icon.spec.tsx
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Icon from './index';

describe('<Icon />', () => {
  it('should display a Icon component', () => {
    const component = renderer.create(<Icon icon="add" />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
