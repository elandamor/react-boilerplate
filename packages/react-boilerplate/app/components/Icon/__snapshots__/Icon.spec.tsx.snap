// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> should display a Icon component 1`] = `
.c0 {
  margin: 0 auto;
  position: relative;
}

.c0 svg {
  vertical-align: inherit;
}

<i
  aria-hidden="true"
  className="c-icon c0"
>
  <svg
    height="24"
    viewBox="0 0 24 24"
    width="24"
  >
    <path
      d="M11 5v6H5v2h6v6h2v-6h6v-2h-6V5h-2z"
    />
  </svg>
</i>
`;
